### PR TITLE
KDESKTOP-1360-Issues-with-Sparkle-updater

### PR DESCRIPTION
**Pull request by ClementKunz**
Once the update has been accepted by the user, the package is downloaded by Sparkle and the user needs to start the update. If another update windows pops up before the validation, the next updates will fail.

Fix : Do not start a new update process if one is already ongoing

### CODE REVIEW

File: src/server/updater/sparkleupdater.mm [hunk 1]
```diff
@@ -187,7 +187,14 @@ - (BOOL)supportsGentleScheduledUpdateReminders {
 }
 
 void SparkleUpdater::onUpdateFound() {
-    if (isVersionSkipped(versionInfo().fullVersion())) return;
+    if (isVersionSkipped(versionInfo().fullVersion())) {
+        LOG_INFO(KDC::Log::instance()->getLogger(), "Version " << versionInfo().fullVersion().c_str() << " is skipped.");
+        return;
+    }
+    if ([d->updater sessionInProgress]) {
+        LOG_INFO(KDC::Log::instance()->getLogger(), "A session is already in progress...");
```
Comment (luc-guyot-infomaniak) [inline src/server/updater/sparkleupdater.mm:195]: Maybe a word about the fact the application is already **updating** itself?

Comment (ClementKunz) [reply src/server/updater/sparkleupdater.mm:196]: Done d7a9ea3030ec854e949b671630d16732b74a9ccc
We are waiting for user input or for the end of an ongoing installation

Comment (luc-guyot-infomaniak) [reply src/server/updater/sparkleupdater.mm:196]: Sorry, I was not precise enough: I thought the log message itself could  mention the ongoing update process, to ease off debugging.

Comment (ClementKunz) [reply src/server/updater/sparkleupdater.mm]: Done. I have removed the comment, no need to say the same thng twice

**Review by ChristopheLarchier — APPROVED** [review]

**Pull request merged**